README.md

```diff
@@ -1,126 +1,100 @@
 ![Official logo](docs/img/Box64Logo.png "Official Logo")
 
-Linux Userspace x86_64 Emulator with a twist
+# Box64: Linux Userspace x86_64 Emulator with a Twist

```

Comment:
Both "x86-64" and "x86_64" refer to the same 64-bit extension of the x86 instruction set architecture, but their usage contexts differ slightly:
* "x86-64" is the original term coined by AMD and is more commonly used in formal technical documentation and academic writing.
* "x86_64" is more frequently used in Unix-like systems, package management, and software development contexts (like in gcc target triplets).

For a technical document, I would recommend using "x86-64" since it's the more formal notation.

Reply:
Sounds Good